tests/compiled_templates/etcdv3/mesh/static-routes/bird_ipam.cfg

```diff
@@ -3,8 +3,8 @@ filter calico_export_to_bgp_peers {
   calico_aggr();
 
   # Export static routes.
-  if ( net = 10.65.1.0/24 ) then { accept; }
-  if ( net = 10.66.1.0/24 ) then { accept; }
+  if ( net = 10.101.0.0/16 ) then { accept; }
```

Comment:
Could optionally do similar here as for `/rejectcidrs` and add a new `/exportcidrs`, but didn't see an immediate need.